### PR TITLE
add buy product scenario

### PR DESCRIPTION
**Pull request by NataliiaBakulieva**
add buy product scenario

### CODE REVIEW

File: tests/buyProduct_test.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+   let sizePrice = await productPage.getPricePerSize();
+
+   I.openCheckoutPage();
+   checkoutPage.fillFieldsCheckout(newAdress);
+
+   let totalPrice = await checkoutPage.getTotalPrice();
+   let shippingRatePrice = await checkoutPage.getFlatShippingRatePrice();
+   let quantity = await checkoutPage.getQuantity();
+
+   price = +price.match(/\d+\.\d+/);
```
Comment (adudyak) [inline tests/buyProduct_test.js]: Use `.replaceAll()` method on string to cleanup price.
Move cleanup method to steps_file, and use it inside of page objects, so in scenario you get clean price.

File: tests/buyProduct_test.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+   quantity = +quantity;
+
+   I.assertEqual((price + colourPrice + sizePrice) * quantity + shippingRatePrice, totalPrice, "Prices are not in match");
+   checkoutPage.purchaseCompletion();
+
+   I.openOrderHistoryPage();
+   let orderID = await orderHistory.getLastOrderID();
+   console.log('Last order ID ' + +orderID.match(/\d+/));
+
+
```
Comment (adudyak) [inline tests/buyProduct_test.js]: remove empty new lines at the end. Between instructions it is ok to have them

File: pages/checkout.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+const { continueBatton } = require("./register");
```
Comment (adudyak) [inline pages/checkout.js]: remove unused import

File: pages/checkout.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    },
+
+    purchaseCompletion() {
+        I.click(this.confirmOrderButton);
+        I.see('Your order has been placed!');
+    },
+
+
+
+
```
Comment (adudyak) [inline pages/checkout.js]: remove new lines

File: tests/buyProduct_test.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+   totalPrice = +totalPrice.match(/\d+\.\d+/);
+   shippingRatePrice = +shippingRatePrice.match(/\d+\.\d+/);
+   quantity = +quantity;
+
+   I.assertEqual((price + colourPrice + sizePrice) * quantity + shippingRatePrice, totalPrice, "Prices are not in match");
+   checkoutPage.purchaseCompletion();
+
+   I.openOrderHistoryPage();
+   let orderID = await orderHistory.getLastOrderID();
+   console.log('Last order ID ' + +orderID.match(/\d+/));
```
Comment (adudyak) [inline tests/buyProduct_test.js]: Move transformation to `getLastOrderID()`. Try `slice()` method on string to get rid of `#`

**Review by adudyak — APPROVED** [review]

File: tests/buyProduct_test.js (hunk 1 above)
Comment (NataliiaBakulieva) [reply tests/buyProduct_test.js]: I did not understand how to correctly derive the price using the method .replaceAll()

**Pull request merged**